addons/mail_group/data/mail_templates.xml

```diff
@@ -1,35 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <template id="mail_group_send_guidelines">
-            <div>
-                <p>Hello <t t-if="member.partner_id" t-esc="member.partner_id.name"/>,</p>
-                <p>Please find below the guidelines of the <t t-esc="group.name"/> mailing list.</p>
-                <p t-esc="group.moderation_guidelines_msg"/>
-            </div>
-        </template>
-
         <template id="mail_group_footer" name="Mail Group: Footer">
-            <div>
-                <span>_______________________________________________</span>
-                <span>Mailing-List: <t t-esc="group_url"/></span>
-                <span>Post to: <t t-esc="mailto"/></span>
-                <span>Unsubscribe: <t t-esc="unsub_url"/></span>
+            <div id="o_mg_message_footer">
+                <p>_______________________________________________</p>
+                <p>Mailing-List: <t t-esc="group_url"/></p>
+                <p>Post to: <t t-esc="mailto"/></p>
+                <p>Unsubscribe: <a t-att-href="unsub_url" t-esc="unsub_label"/></p>
             </div>
         </template>
 
         <template id="mail_group_notify_moderation">
-            <div>
-                <p>Hello <t t-esc='record.name'/></p>
-                <p>You have messages to moderate, please go for the proceedings.</p><br/><br/>
-                <div style="text-align: center;">
-                    <a href="/web#action=mail_group.mail_group_message_action" class="o_default_snippet_text">Moderate Messages</a>
-                </div>
+            <div style="max-width: 600px">
+                <p>Hello <t t-esc="moderator.partner_id.name"/>,</p>
+                <p>You have messages to moderate, please go for the proceedings.</p>
+                <p><a t-attf-href="/web#action=mail_group.mail_group_action&amp;id={{group.id}}&amp;view_type=form" class="o_default_snippet_text">Moderate Messages</a></p>
                 <p>Thank you!</p>
             </div>
         </template>
     </data>
 
+    <record id="mail_template_guidelines" model="mail.template">
+        <field name="name">Mail Group: Send Guidelines</field>
+        <field name="model_id" ref="mail_group.model_mail_group_member"/>
+        <field name="subject">Guidelines of group ${object.mail_group_id.name}</field>
+        <field name="email_to">${object.email | safe}</field>
```

Comment:
The will be big because it should take in order
- email_formated of the company of the partner
- catchall_formatted of the company of the partner
- email_formated of the current company
- catchall_formatted of the current company